14/index.php

```diff
@@ -0,0 +1,174 @@
+<?php
+require_once('includes/template.php');
+require_once('includes/ui/downloads.php');
+require_once('includes/appstore.php');
+require_once('includes/playstore.php');
+
+// Required
+head([
+  'class' => 'keyman11',
+  'title' => 'Keyman 14 is now in Beta!',
+  'css' => ['template.css', 'dev.css', 'app-store-links.css', 'prism.css'],
+  'js' => ['prism.js'],
+  'showMenu' => true,
+  'banner' => [
+    'title' => 'Keyman 14.0 Beta',
+    'image' => 'world.png',
+    'background' => 'water'
+  ]
+]);
+?>
+
+<h2>Announcing the release of Keyman 14.0 Beta</h2>
+<p class="red">30 November 2020</p>
+<p>We are pleased to announce that Keyman 14.0 is now in beta!</p>
+<br>
+
+<h3>What's New?</h3>
+<ul>
+  <li>
+    Simpler and Smoother Keyboard Seach<br>
+    <img alt='Search for Khmer Keyboard' src='keyboard_search_khmer.png'>
+  </li>
+  <li>
+    Localizable UI through <a href="https://translate.keyman.com">translate.keyman.com</a>.
+  </li>
+  <li>
+    Mobile apps download and install keyboard packages from keyman.com
+  </li>
+  <li>
+    Consolidated crash reporting to <a href="https://sentry.keyman.com">sentry.keyman.com</a>
```

Comment:
```suggestion
    Consolidated crash reporting to <a href="<?= KeymanHosts::sentry_keyman_com ?>">sentry.keyman.com</a>
```

And we'll need to add sentry_keyman_com to KeymanHosts.